Founta2/chat gpt

Fountai2 opened the pull request:
Super early version, just to get the weirdly nasty api out. 

Plan to add more and clean it up as I go.

Currently does team lookup but I want to remove the case sensitivity (right now you need to chirp a case sensitive name since the name in the Dict in capitalized) but I also want to pull in game objects to look up prior game scores and maybe even standings.

Commit:
chatgpt chirps

Fountai2 committed:
Adding in a chatgpt chirp bot that should be less manic and potentially vile.

You'll need an openAI key for your credentials file ping me for it.

open_ai_key = ""

jschultz38 commented on bot.py:
```diff
@@ -178,6 +178,18 @@ async def chirp(ctx, user=None):
     async def ruf(ctx):
         await ctx.send('https://imgur.com/q4OWXNs')
 
```
Looks like you created an extra chirp command? Maybe change the command name to something that people can't figure out so we can continue to use it for testing purposes?

Fountai2 replied:
Ah I may have failed some cleanup (I originally had a duplicate command called "chirpgpt")

Fountai2 replied:
Weird, this is not what's on my local...

Fountai2 committed:
This should clean up the chirp command.

jschultz38 committed:
Clean up PR

Update readme to include new credentials and remove a
'fuck off' print lol

also merge with latest

Signed-off-by: Jarrett Schultz <jschultz38@gatech.edu>
